markdown/api-docs/cart-and-checkout/add-to-cart-url.md

```diff
@@ -1,89 +1,92 @@
+```html
+<a href="https://example.com/cart.php?action=buy&product_id=123&source=blogpost">Purchase Our New Product Now!</a>
+```
 
-### Structure:
-`/site.com/cart.php?action=add&sku=INSERT-SKU-HERE`
+## Adding Multiple Products
 
-### Example:
-`myawesomestore.com/cart.php?action=add&sku=SHIRT-SM-RED`
+The `sku` and `product_id` parameters accept a single value; if a comma seperated list of values is passed in, only the first value is used. In other words, only one product can be added for each request made to an add to cart URL; however, its possible to combine several HTTP requests into a single button click using front-end javascript. 
 
----
+Here's a very basic example using jQuery:
 
-<a href='#add-cart-url_add-sku-checkout-go-to-cart' aria-hidden='true' class='block-anchor'  id='add-cart-url_add-sku-checkout-go-to-cart'><i aria-hidden='true' class='linkify icon'></i></a>
+```html
 
-## Add Specific SKU to Cart and Go Directly to Checkout
-To automatically add a product or variant to a shopper's cart and forward them directly to checkout, append   
-`/cart.php?action=buy&sku=INSERT-SKU-HERE` to the store's domain.
+<button type="button" id="addToCart">Add Bundle to Cart</button>
 
-### Structure:
-`/site.com/cart.php?action=buy&sku=INSERT-SKU-HERE`
+<script>
+// when #addToCart is clicked...
+$("button#addToCart").click(function() {
 
-### Example:
-`myawesomestore.com/cart.php?action=buy&sku=SHIRT-SM-RED`
+	// add product id 123
+    $.get("/cart.php?action=add&product_id=123");
 
----
+	// add product id 456
+    $.get("/cart.php?action=add&product_id=456", function(data) {
 
-<a href='#add-cart-url_add-specific-sku-checkout-source' aria-hidden='true' class='block-anchor'  id='add-cart-url_add-specific-sku-checkout-source'><i aria-hidden='true' class='linkify icon'></i></a>
+		// go to cart
+		window.location = "/cart.php";
+	});
+});
+</script>
+```
 
-## Add Specific SKU, Go to Checkout, and Include Source
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+    
+<!-- theme: warning -->
 
-To automatically add a product or variant to a shopper's cart, forward them to checkout, and also include a source parameter for analytics/conversion tracking, append `cart.php?action=buy&sku=INSERT-SKU-HERE&source=SOURCE-HERE` to the store's domain. (The&#160;source parameter can be any string.)
+> Due to CORs, using javascript to make multiple carting requests only works in the BigCommerce storefont and only on the storefront with the domain the request is being made to. 
 
-### Structure:
-`site.com/cart.php?action=buy&sku=INSERT-SKU-HERE&source=SOURCE-HERE`
+Alternitively, The [Storefront Cart API's](https://developer.bigcommerce.com/api-docs/cart-and-checkout/working-sf-apis#working-sf-apis_storefront-cart) `/api/storefront/cart` endpoint accepts an array of `lineItems` -- depdending on the complexities and specifics of the use case, this may be a better solution than add to cart urls. 

```

Comment:
> depdending on the complexities and specifics of the use case, this may be a better solution than add to cart urls. 
```suggestion
Alternitively, The [Storefront Cart API's](https://developer.bigcommerce.com/api-docs/cart-and-checkout/working-sf-apis#working-sf-apis_storefront-cart) `/api/storefront/cart` endpoint accepts an array of `lineItems` -- depending on the complexities and specifics of the use case, this may be a better solution than add to cart URLs. 
```

Reply:
Forgot to say, I am cool with approving this once the suggestions are fixed.